zip-0227.rst

```diff
@@ -337,13 +312,18 @@ If all of the above checks pass, do the following:
 - If :math:`\mathsf{finalize} = 1`, add :math:`\mathsf{AssetDigest}` to the :math:`\mathsf{previously\_finalized}` set immediately after the block in which this transaction occurs.
 - (Replay Protection) If issue bundle is present, the fees MUST be greater than zero.
 
+Rationale for non-zero Issue Bundle fees
+----------------------------------------
+We require non-zero fees in the presence of an issue bundle, in order to preclude the possibility of a transaction containing only an issue bundle. 
```

Comment:
"in order to preclude the possibility of a replay attack."

Then, describe the potential attack.

Reply:
Done